lazer/sdk/js-solana/src/idl/pyth-lazer-solana-contract.json

```diff
@@ -0,0 +1,464 @@
+{
```

Comment:
If these files are generated, I suggest not checking them in.  Instead, ensure the build tasks are correctly ordered so the files get generated as part of building this package.  I personally really prefer not to check these files in, since it is effectively a cache and checking them in means you are now susceptible to cache sync issues.